docs/development-guide/ReadWriteSplittingPluginPerformanceResults.md

```diff
@@ -6,17 +6,3 @@
 | No Plugin Enabled                                             | 1,105,514.4 | ms/op |
 | Read-Write Splitting Plugin Enabled                           | 1,106,074.0 | ms/op |
 | Read-Write Splitting Plugin and Reader Load Balancing Enabled | 1,107,379.9 | ms/op |
-
```

Comment:
Why are we removing these ?

Reply:
We are investigating the numbers. The read-write plugin implementation is not finalized yet so we will add these performance results later